pkg/util/iswimi/iswimi.go

```diff
@@ -0,0 +1,14 @@
+package iswimi
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import "github.com/Azure/ARO-RP/pkg/api"
+
+// IsWimi checks whether a cluster is Workload Identity or classic
+func IsWimi(cluster api.OpenShiftClusterProperties) bool {
```

Comment:
nit: I would make this argument `OpenShiftCluster`, not `OpenShiftClusterProperties`.
It would make a bit more sense to me and it accords with the argument name.

Reply:
Super nit - I think we should also use `oc` as the param name instead of `cluster` since it's usually what we use.

Reply:
done, thank you!